_posts/2019-01-23-cockpit-186.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 186
+author: Katerina Koukiou
+date: 2019-01-23
+tags: cockpit linux
+slug: cockpit-186
+category: release
+summary: Responsive system page and improvement in services page
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 186.
+
+### Services: The Services page has been redesigned to make it easier to understand
+
```

Comment:
This was in relation to the usability testing we did, it would make sense to mention that this is happening as a result of that.

Reply:
Done

Reply:
Can we shorten the heading a bit, and move that to the text? Maybe "Redesign Services page".